ddtrace/internal/_encoding.pyx

```diff
@@ -278,15 +282,15 @@ cdef class MsgpackEncoder(MsgpackEncoderBase):
         finally:
             self._reset_buffer()
 
-    cdef inline int _pack_meta(self, object meta, object dd_origin):
+    cdef inline int _pack_meta(self, object meta, char *dd_origin):
         cdef Py_ssize_t L
         cdef int ret
         cdef dict d
 
         if PyDict_CheckExact(meta):
             d = <dict> meta
             L = len(d)
-            if dd_origin is not None:
+            if dd_origin is not NULL:
```

Comment:
I wonder how this gets converted, should it be `if dd_origin != NULL` ?

Reply:
Nope, the docs here reference being able to directly use the `is` operator with `NULL`: https://cython.readthedocs.io/en/latest/src/userguide/language_basics.html#checking-return-values-of-non-cython-functions

Reply:
cool, great